examples/basic_diesel/.env

```diff
@@ -0,0 +1 @@
+DATABASE_URL=.posts.db
```

Comment:
Suggest we hardcode this for the example, goal being to streamline 3rd party overheads/concepts as much as possible.

One of the examples I have planned is going to give some ideas for external configuration loading so we'll cover this then.